crowbar_framework/app/models/service_object.rb

```diff
@@ -1400,6 +1400,15 @@ def apply_role_post_chef_call(old_role, role, all_nodes)
     # noop by default.
   end
 
+  # This callback provides a hook into the schema migration procedure.
+  # It is called from lib/schema_migration.rb after the schema migrations
+  # for a proposal (and its role, if present) have been executed. Parameters
+  # are the migrated proposal an role objects. The default implementation is
+  # a noop. (To be overwritten in sub classes)
+  def post_schema_migration_callback(proposal, role)
+    # noop by default
```

Comment:
maybe just do a `return`?

Reply:
@rsalevsky Why? Does that change anything? (I'd prefer to keep it like that, because the two methods above are actually defined in  the very same way)

Reply:
@rhafer Technically it's the same but it would be the usual way how it is done in ruby.